### ISSUE TITLE
Issue with logging file location

### ISSUE THREAD

**Issue by winkste** [AppDaemon/appdaemon#1166]
Hi.

I'm new to the appdeamon community. Currently I was pretty successful switching my automation from pure HASSIO scripting to python appdeamon scripting. Unfortunately I'm struggling with two issues. The first one is, that I tried to configure the file logging as described in several tutorials, but I can't find the files. 
I'm running Appdeamon 4 in version 0.4.3.
My configuration is:
<img width="522" alt="Bildschirmfoto 2021-02-18 um 06 22 02" src="https://user-images.githubusercontent.com/9803344/108309338-a8cba300-71b1-11eb-8fc8-8b39ca104895.png">

I would expect files located in:
/root/config/appdaemon/logs

Restarting appdeamon seems to work correct for logging:
<img width="731" alt="Bildschirmfoto 2021-02-18 um 06 26 39" src="https://user-images.githubusercontent.com/9803344/108309678-458e4080-71b2-11eb-87e7-1a762c35c529.png">

I'm also wondering that I don't see the same configuration in the WebUi than in the appdeamon.yaml file?
<img width="428" alt="Bildschirmfoto 2021-02-18 um 06 28 44" src="https://user-images.githubusercontent.com/9803344/108309818-930aad80-71b2-11eb-917d-90d84b091d6d.png">

But I don't see them in SSH as in a SAMBA connection. Can someone help?

**Comment by ReneTode** [comment]
your log configuration should be in appdaemon.yaml

**Comment by frenck** [comment]
You are editing the add-on configuration indeed, however, those configuration options are part of AppDaemon itself, not the add-on configuration.

The AppDaemon configuration can be found in `/config/appdaemon/AppDaemon.yaml`, which you can access e.g., via Samba, SSH, FTP, or any of the file editors available.

👍

**Comment by winkste** [comment]
Ok, first of all: Thanks for the fast response. I already wondered the two different configurations. 

My log folder shall be:
<img width="223" alt="Bildschirmfoto 2021-02-18 um 20 57 24" src="https://user-images.githubusercontent.com/9803344/108413939-e9680280-722b-11eb-8a2f-8be40fb91b87.png">

I moved the log configuration to the AppDaemon.yaml file as follows:
<img width="434" alt="Bildschirmfoto 2021-02-18 um 20 55 18" src="https://user-images.githubusercontent.com/9803344/108413706-a017b300-722b-11eb-96b3-818b5f6b8e32.png">

Currently I got the response:
<img width="727" alt="Bildschirmfoto 2021-02-18 um 20 53 12" src="https://user-images.githubusercontent.com/9803344/108413443-53cc7300-722b-11eb-8976-153d7ef23f9f.png">

It looks like, AppDaemon is not able to find the location or it doesn't have access rights to the folder. AppDaemon is repeating this several times and than stops processing.
